### DISCUSSION TITLE
Any way to get the song url through the mpris interface?

### DISCUSSION THREAD

**Discussion by QuditWolf** [pear-devs/pear-desktop#1135] [Q&A]
Title says all :)

I want this because I want to add a listen along link where I embed what I'm currently listening to. (on youtube-music)
I use playerctl to query the mpris metadata.

Thx

**Comment by QuditWolf** [comment]
@itzmanish I think you did the mpris plugin... how tough would that be?

Pinging you. hope it's not against etiquette.

**Reply by itzmanish** [reply]
It should not be tough. You only need to return url in the metadata when you are queried for metadata from playerctl. You can check the handler where mpris plugin handles the different request of playerctl.

**Reply by QuditWolf** [reply]
okay, added that property to mpris.js and have made a pull request. (might be useful to others too)
for reference: https://github.com/th-ch/youtube-music/pull/1138
